src/cryptography/x509/base.py

```diff
@@ -592,7 +592,7 @@ def add_extension(
         )
 
     def add_attribute(
-        self, oid: ObjectIdentifier, value: bytes
+        self, oid: ObjectIdentifier, value: bytes, tag: int = 12
```

Comment:
We have https://github.com/pyca/cryptography/blob/ccfef6f1076c9581e7641d52664ce90de0fd7e0a/src/cryptography/x509/name.py#L16 so you don't need to have it be an integer.

Reply:
Are you suggesting the argument have that type, or just that we use it to get the UTF8String constant?

Reply:
I think we should use the type itself and probably make it _tag as well

Reply:
that class misses quite a thing like octetstring